packages/starkiller/PKGBUILD

```diff
@@ -0,0 +1,53 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=Starkiller
```

Comment:
pkgname should be lowercase, it's not an issue for the URL since github url are case incensitive